config/routes.rb

```diff
@@ -43,7 +43,10 @@
 
     resources :organization_members, as: 'members', only: %i[index edit update destroy]
     resources :service_specifications, except: :show
-    resources :service_agreements, only: %i[index destroy]
+    resources :service_agreements, only: %i[index destroy create]
+    get '/service_agreements/civil_servants/search',
+        to: 'service_agreements#search', as: 'service_agreement_civil_servant_search'
+    match '/service_agreements/new', to: 'service_agreements#new', via: %i[get post]
```

Comment:
I don't get the match here, why not just add new and post to the `resources :service_agreements`?

Reply:
I had trouble with it listening to post and get